dotCMS/src/main/java/com/dotcms/rest/api/v1/container/ContainerResource.java

```diff
@@ -355,7 +370,14 @@ private String getHTML(final HttpServletRequest req,
         context.put("contentletList" + container.getIdentifier() + Container.LEGACY_RELATION_TYPE,
                 Lists.newArrayList(contentlet.getIdentifier()));
         context.put(mode.name(), Boolean.TRUE);
-        context.put("dotPageContent",Boolean.TRUE);
+
+        if (UtilMethods.isSet(pageInode)) {
```

Comment:
I think it would be better to leave this to the PageLoader?  It does other things that the context might want, like including the host and host variables.   Maybe do something like:
```
final VelocityResourceKey pageKey = new VelocityResourceKey(htmlPage, mode, language);
return velocityUtil.merge(pageKey.path, context);
```

Reply:
it is really not working just when the ContainerResource end point